sdk-scheme-adapter/chart-service/templates/deployment.yaml

```diff
@@ -63,6 +63,17 @@ spec:
         {{- if .Values.initContainers }}
           {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
         {{- end }}
+      volumes:
```

Comment:
Please could we add a "checkum" to the deployment annotations to make sure that the container will be restarted if the secrets are changed, i.e. have a look at this examples 
1. https://github.com/mojaloop/helm/blob/master/ml-api-adapter/chart-handler-notification/templates/deployment.yaml#L28
2. https://github.com/mojaloop/helm/blob/master/mojaloop-simulator/templates/deployment.yaml#L34

Reply:
Done